test/integration/analyze_test.cpp

```diff
@@ -0,0 +1,492 @@
+
+#include "execution/compiler/output_checker.h"
+#include "integration/end_to_end_test.h"
+#include "optimizer/statistics/histogram.h"
+#include "optimizer/statistics/top_k_elements.h"
+#include "spdlog/fmt/fmt.h"
+#include "test_util/test_harness.h"
+
+namespace noisepage::test {
```

Comment:
Let me know if you have any thoughts on the placement of this file

Reply:
Maybe a `sql` directory, again no strong thoughts though.

Reply:
@thepinetree what do you mean by a `sql` directory?

Reply:
Moved